tests/api_tests/nodes/test_views.py

```diff
@@ -1349,3 +1351,115 @@ def test_return_deleted_private_node_pointer(self):
         #check that deleted pointer can not be returned
         res = self.app.get(self.private_url, auth=self.basic_auth, expect_errors=True)
         assert_equal(res.status_code, 404)
+
+
+class TestNodeLogList(ApiTestCase):
+
+    def setUp(self):
+        super(TestNodeLogList, self).setUp()
+        self.user = UserFactory.build()
+        password = fake.password()
+        self.logs = NodeLogFactory.build()
+        self.password = password
+        self.user.set_password(password)
+        self.user.save()
+        self.basic_auth = (self.user.username, password)
+
+        self.user_two = UserFactory.build()
+        self.user_two.set_password(self.password)
+        self.user_two.save()
+        self.add = ProjectFactory(is_added=True)
+        self.basic_auth_two = (self.user_two.username, self.password)
+        self.private_project = ProjectFactory(is_public=False, creator=self.user)
+        self.private_url = '/{}nodes/{}/logs/'.format(API_BASE, self.private_project._id)
+        self.public_project = ProjectFactory(is_public=True, creator=self.user)
+        self.public_url = '/{}nodes/{}/logs/'.format(API_BASE, self.public_project._id)
+
+    def assert_datetime_equal(dt1, dt2, allowance=500):
+        assert_less(dt1 - dt2, dt.timedelta(milliseconds=allowance))
+
+    def test_log_create_on_public_project(self):
+        datetime.datetime.strptime('2015-07-28 21:06:34.965114','%Y-%m-%d %H:%M:%S.%f')
+        date_str = "2008-11-10 17:53:59"
+        dt_obj = datetime.datetime.strptime(date_str, "%Y-%m-%d %H:%M:%S")
+        print repr(dt_obj)
```

Comment:
This looks like code that was for trying to figure out how this works, rather than code you're actually using for the test. If this code isn't necessary for the test, please remove it. If it is necessary, what is it doing?